release-conf.yaml

```diff
@@ -1,7 +1,3 @@
-# list of major python versions that release-bot will build separate wheels for
-python_versions:
-  - 3
-
```

Comment:
Please, leave that here until the new release of release bot..;-)

Reply:
We would like to get rid of python-2 support in this sprint. We will release release-bot w/o python2 support soon.

Reply:
Then LGTM.